document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresDefaults.java

```diff
@@ -3,11 +3,8 @@
 import java.time.Duration;
 
 public interface PostgresDefaults {
-  String DEFAULT_USER = "postgres";
```

Comment:
Not sure if these were refactored or removed - any removal of default behavior would be a breaking change we should avoid.